packages/oidc-client/src/lib/authorize.request.utils.ts

```diff
@@ -131,11 +197,19 @@ export function createAuthorizeErrorµ(
   );
 }
 
+/**
+ * @function createAuthorizeUrlµ
+ * @description Creates an authorization URL and related options/config for the Authorize request.
+ * @param {string} path - The path to the authorization endpoint.
+ * @param { OidcConfig } config - The OIDC client configuration.
+ * @param { GetAuthorizationUrlOptions } options - Optional parameters for the authorization request.
+ * @returns { Micro.Micro<[string, OidcConfig, GetAuthorizationUrlOptions], AuthorizeErrorResponse, never> }
+ */
 export function createAuthorizeUrlµ(
   path: string,
   config: OidcConfig,
   options: GetAuthorizationUrlOptions,
-) {
+): Micro.Micro<[string, OidcConfig, GetAuthorizationUrlOptions], AuthorizeErrorResponse, never> {
```

Comment:
Nothing wrong here. Just wanted to mention that I really like when you include the return types for effects on functions like you did here. It helps me better understand the return type without having to hover over for intellisense.